index.js

```diff
@@ -0,0 +1,72 @@
+import AwesomeBooks from './modules/awesome.js';
+import luxon from './modules/luxon.js';
+
+const awesomeBooks = new AwesomeBooks();
+awesomeBooks.load();
+if (awesomeBooks.getBooks().length < 1) {
+  awesomeBooks.addBook('The Hobbit', 'J.R.R. Tolkien');
+  awesomeBooks.addBook('hello JS', 'ricky ricky');
+}
+
+const bookListSection = document.querySelector('#book-list');
+
+function renderBook() {
+  bookListSection.innerHTML = awesomeBooks.getBooks().map((book, index) => `
+        <article class="book ${index % 2 === 0 ? 'dark' : ''}">
+            <div>
+                <p class="title">"${book.title}" by ${book.author}</p>
+            </div>
+            <div class="button-remove">
+            <button data-id=${book.id} class="remove">Remove</button>
+            </div>
+        </article>`).join('');
+}
```

Comment:
- [ ] Please, consider to use ES6 syntax, aka array functions. Here you can use
```
const renderBook = () => {
.....
}
```

Reply:
@elmar8287 
thank you so much fixed the issues